### ISSUE TITLE
Battle works fine but doesn't select or scroll support

### ISSUE THREAD

**Issue by SleepyAhn** [MathewSachin/Fate-Grand-Automata#420]
> MAKE SURE YOU TESTED THE LATEST RELEASE.

> Did you look at other issues and the [Troubleshooting Guide](https://github.com/MathewSachin/Fate-Grand-Automata/wiki/Troubleshooting)?

**Describe the bug**
Skills order, NP and battle all works fine
It's just that I can't loop because FGA doesn't select a support
I tried to full screen one or the other or both together and it doesn't select click/scroll support either way
FGA only works in battle if both are not full screen too 

**Screenshots**
It is recommended that you atleast add one screenshot of FGO running on your device.
These are my settings 

![Screenshot_20200917-142823_FateGrand_Automata](https://user-images.githubusercontent.com/71425793/93428655-6ea06180-f8f2-11ea-991b-fd6113aad09e.jpg)
![Screenshot_20200917-142905_FateGrand_Automata](https://user-images.githubusercontent.com/71425793/93428661-6fd18e80-f8f2-11ea-94a2-118b38315557.jpg)

**Video**
I'm the one who clicked on the support myself at the beginning of the video because FGA doesn't

https://www.youtube.com/watch?v=93Tybnk9n3U&ab_channel=SleepyAran

**Device Info (please fill atleast 'Device model'):**
 - Device model:  Samsung Galaxy S10 Lite
 - Screen size: 1080x2400
 - Android Version: Android 10
 - RAM: 8 RAM

**Comment by MathewSachin** [comment]
First line in the issue literally says this: Did you look at other issues and the Troubleshooting Guide?

**Comment by SleepyAhn** [comment]
Hi, sorry. Did I look at the troubleshooting guide? Yeah, that's why I tried to change around the full screen thing like you wrote in there with the Samsung settings and  it does give blue screens at the side but than FGA stops working. The other issues you mainly told them to go read the same guide about the screen detection area / turning on or off the notch detection or download the latest release but I just downloaded it from google play an hour ago so I just assume it's the latest(?). Sorry I'm really stupid

**Comment by MathewSachin** [comment]
I mean the video you uploaded didn't have blue bars, the area is black, so how would I know?

**Comment by reconman** [comment]
Can you post a video of blue bars not working with debug mode?

**Comment by reihunter360** [comment]
BUG/Problem:
I've got a similar problem, but i think it roots off from FGA detecting the FateGOJP clone (its com.aniplex.fategrandordes) as FGONA.
Under More Options; I've set it back to JP Mode, but it keeps switching back to NA Mode.
It can select the node/ go into battle just fine, but not the Support Selection Screen. It just gets stuck there in support screen. 
It was working this morning too.
I haven't changed any settings anywhere, except for Left Padding for the Screen Size (0 for default, 90 when in use). That's for the past few days.

As for my regular FGOJP and FGONA clients, FGA is working fine.

VIDEO: https://youtu.be/0yUvBQuPyJc

Note : I'm the one selecting the classes tab, to get a reaction out of FGA's detections.
The autoskill script is programmed to pick the first servant of MIX.

Device Info :
Phone : Nokia 5 (no notch~)
Screen Size : 720x1280
Android Ver : Android 8.1 (64bit ver.)
RAM : 2GB
